spec/controllers/events_controller_spec.rb

```diff
@@ -1,5 +1,46 @@
 require 'rails_helper'
 
 RSpec.describe EventsController, type: :controller do
+ 	
+ 	let(:user) {User.create(email: "tester@test.com", password: "1234")}
+  let(:valid_params) {{ name: "iOS Pitch", date: "1/1/2001", start_time: '16:00', end_time: '18:00', user_id: user.id }}
+  let(:invalid_params) {{ name: "Invalid Pitch", description: "date and time not provided" }}
+
+  describe "GET #new" do
+    before do
+      session[:user_id] = user.id
```

Comment:
Since you've added `clearance/rspec` in `rails_helper`, you should be able to do `sign_in user` rather than `session[:user_id] = user.id`. 
Clearance comes with a few useful helper methods for testing, can refer to [https://github.com/thoughtbot/clearance#controller-test-helpers](https://github.com/thoughtbot/clearance#controller-test-helpers)